public/static/locales/en/landing.json

```diff
@@ -1,7 +1,7 @@
 {
   "home": {
     "banner": {
-      "heading": "Find a travel insurance provider</br> if you have a serious medical condition",
+      "heading": "Find a travel insurance provider if you have a serious medical condition",
```

Comment:
I don't think we can just remove the `</br>` tag without a CSS change. If I understand correctly the requirement is to constrain the width of the heading but my feeling is we need to do that by a different method so that the text looks acceptable at all widths.